src/Swarm/TUI/View.hs

```diff
@@ -261,10 +286,10 @@ generateModal s mt = Modal mt (dialog (Just title) buttons (maxModalWindowWidth
       WinModal -> ("", txt "Congratulations!", Nothing, maxModalWindowWidth)
       DescriptionModal e -> (descriptionTitle e, descriptionWidget s e, Nothing, 100)
       QuitModal ->
-        let quitMsg = "Are you sure you want to quit?"
+        let quitMsg = "Are you sure you want to quit this game and return to the menu?"
          in ( ""
             , padBottom (Pad 1) $ hCenter $ txt quitMsg
-            , Just (0, [("Cancel", Cancel), ("Quit", Confirm)])
+            , Just (0, [("Keep playing", Cancel), ("Quit to menu", Confirm)])
```

Comment:
Actually, my request already has an Issue:
- #25

Reply:
> could it take me to the parent scenario folder instead of back to the main menu?
> 
> Maybe that could be in future PR. slightly_smiling_face

Yeah, I don't have anything against that in principle, but let's leave it to a future PR.